<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>701</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt </string>
    </property>
    <property name="text">
     <string>The size of the structural element:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="strcutElemDim">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="chooseFileBtn"/>
   <addaction name="separator"/>
   <addaction name="erodeImageBtn"/>
   <addaction name="separator"/>
   <addaction name="dilateImageBtn"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="squareShape"/>
   <addaction name="separator"/>
   <addaction name="rhombusShape"/>
   <addaction name="separator"/>
   <addaction name="crossShape"/>
   <addaction name="separator"/>
   <addaction name="xShape"/>
   <addaction name="separator"/>
  </widget>
  <action name="chooseFileBtn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/dir.png</normaloff>:/icons/dir.png</iconset>
   </property>
   <property name="text">
    <string>Choose file</string>
   </property>
   <property name="toolTip">
    <string>Choose file</string>
   </property>
  </action>
  <action name="squareShape">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/square.png</normaloff>:/icons/square.png</iconset>
   </property>
   <property name="text">
    <string>Square</string>
   </property>
   <property name="toolTip">
    <string>Square</string>
   </property>
  </action>
  <action name="rhombusShape">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rhombus.png</normaloff>:/icons/rhombus.png</iconset>
   </property>
   <property name="text">
    <string>Rhombus</string>
   </property>
   <property name="toolTip">
    <string>Rhombus</string>
   </property>
  </action>
  <action name="crossShape">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
   </property>
   <property name="text">
    <string>Cross</string>
   </property>
   <property name="toolTip">
    <string>Cross</string>
   </property>
  </action>
  <action name="xShape">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/x.png</normaloff>:/icons/x.png</iconset>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="toolTip">
    <string>X</string>
   </property>
  </action>
  <action name="erodeImageBtn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/erode.png</normaloff>:/icons/erode.png</iconset>
   </property>
   <property name="text">
    <string>Erode</string>
   </property>
   <property name="toolTip">
    <string>Erode image</string>
   </property>
  </action>
  <action name="dilateImageBtn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/dilate.png</normaloff>:/icons/dilate.png</iconset>
   </property>
   <property name="text">
    <string>Dilate</string>
   </property>
   <property name="toolTip">
    <string>Dilate image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
